pkg/bbgo/exit_protective_stop_loss.go

```diff
@@ -30,15 +30,19 @@ type ProtectiveStopLoss struct {
 	// PlaceStopOrder places the stop order on exchange and lock the balance
 	PlaceStopOrder bool `json:"placeStopOrder"`
 
+	// Interval is the time resolution to update the stop order
+	// KLine per Interval will be used for updating the stop order
+	Interval types.Interval `json:"interval,omitempty"`
```

Comment:
can you set the default value to 1m if it’s empty?

Reply:
ok